### ISSUE TITLE
Simplify or remove 'related memoranda'

### ISSUE THREAD

**Issue by mathewcropper** [GSA/plainlanguage.gov#158]
## Summarize the issue in one sentence

On the [home page](https://plainlanguage.gov) the _law and requirements_ section contains the phrase _related memoranda_, which is not a phrase that many will understand.

## Tell us more about how we can help

Simplify the wording of this section summary, or remove the phrase entirely if you find that it is not essential.

## Details

* Related links - n/a
* Operating system and version (example: Windows 10 or Mac OS X 10.12.6) - n/a
* Browser and version (example: Chrome, Internet Explorer, or Safari) - n/a

**Comment by nicoleslaw** [comment]
@mathewcropper Hello, thanks for your suggestion! Would `related memos` work for you?

**Comment by mathewcropper** [comment]
Hey. Thanks for the fast response!

I think it depends on whether you think that your average user will understand what a memo is? I'd be tempted to abstract memos and exec orders, and call them "related documents" or something similar?

Your call entirely.. I certainly don't have the perfect answer. It just jumped out of the page as being a difficult term.

**Comment by nicoleslaw** [comment]
Sweet—thank you! I will do a bit of editing there. The site is primarily for federal employees but we know other people reference it too 👍
